src/index.html

```diff
@@ -2,12 +2,35 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__image" src="./images/imac.jpeg"
+        alt="iMac">
+      <div class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__rank">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews"> Reviews: 5 </p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-name"> Price: </p>
+        <p class="card__price-cost"> $2,199 </p>
```

Comment:
```suggestion
        <p class="card__price-name"> Price: </p>
        <p class="card__price-amount"> $2,199 </p>
```
amount sounds better - don't you think?